src/components/form/form-question.tsx

```diff
@@ -1,6 +1,8 @@
 import { FormColumn } from '@/types/form'
 import { UIDataType } from '@/types/ui-data'
 
+import { Input } from '../input'
```

Comment:
nit: personally, I prefer
```suggestion
import { Input } from '@/components/input'
```